### ISSUE TITLE
PlaygameServices Login is not working id i am using enablesavegames feature what should i do ?

### ISSUE THREAD

**Issue by EasybotGames** [playgameservices/play-games-plugin-for-unity#2483]
When i am try to sign in without enablesavegame everything works fine but when i used it i am unable to login.I am attaching the log file if it helps. Please reply soon my project is on halt because of this.

[log.zip](https://github.com/playgameservices/play-games-plugin-for-unity/files/2929369/log.zip)

**Comment by 0c0de** [comment]
I'm on the same as you, your only option is to store that data on a server because of security things, or made locally and deal with possible crackers changing data in files

**Comment by EasybotGames** [comment]
Is this is known to google officials if yes they are doing something about it or not ?

**Comment by Vonak** [comment]
Status: Won't Fix (Intended Behavior)
Please find the below comments from the development team.

"We are going to support the Saved Games in Play Games services, and implement the migration internally inside the Play Games services, so the API will no change for the developers.
Except, possibly, for the SCOPE_APPFOLDER, we are planing to introduce a copy of this constant in Play Games services, maybe with a with a different name, e.g. SCOPE_SAVEDGAMES.
To summarise: no migration required from the developers at the moment."

source: https://issuetracker.google.com/issues/121345407

**Comment by EasybotGames** [comment]
but their official said this thread is not for issue we are talking

**Comment by ProtoJerry** [comment]
A workaround is available until it's fixed #2429

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
